llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/TransformOps/LinalgExtTransformOps.td

```diff
@@ -105,14 +105,50 @@ def RewriteForallToScfForOp :
   }];
 }
 
-def TileAndDecomposeAttentionOp : Op<Transform_Dialect, "tile_and_decompose_attention",
+def TileAttentionOp : Op<Transform_Dialect, "tile_attention",
     [FunctionalStyleTransformOpTrait,
      MemoryEffectsOpInterface,
      TransformOpInterface,
      TransformEachOpTrait,
      ReportTrackingListenerFailuresOpTrait]> {
   let description = [{
-    Target iree_linalg_ext.attention ops and tile and decompose them.
+    Target iree_linalg_ext.attention ops and tile them.
+    This transform consumes the target handle and produces a result handle.
+  }];
+
+  let arguments = (
+      ins TransformHandleTypeInterface:$target
+  );
+  let results = (outs Variadic<TransformHandleTypeInterface>:$result);
+
+  let assemblyFormat = "attr-dict $target `:` functional-type(operands, results)";
+  let cppNamespace = "mlir::iree_compiler::IREE::LinalgExt";
+
+  let builders = [
+    OpBuilder<(ins "Value":$target)>
+  ];
+
+  let assemblyFormat = [{
+    $target attr-dict `:` functional-type(operands, results)
+  }];
+
+  let extraClassDeclaration = [{
+    ::mlir::DiagnosedSilenceableFailure applyToOne(
+        ::mlir::transform::TransformRewriter &rewriter,
+        ::mlir::iree_compiler::IREE::LinalgExt::AttentionOp target,
+        ::mlir::transform::ApplyToEachResultList &results,
+        ::mlir::transform::TransformState &state);
+  }];
+}
+
+def DecomposeTiledAttentionOp : Op<Transform_Dialect, "decompose_tiled_attention",
+    [FunctionalStyleTransformOpTrait,
+     MemoryEffectsOpInterface,
+     TransformOpInterface,
+     TransformEachOpTrait,
+     ReportTrackingListenerFailuresOpTrait]> {
+  let description = [{
+    Target iree_linalg_ext.attention ops and decompose them.
     This transform consumes the target handle and produces a result handle.
```

Comment:
I think this has been a theme with IREE transform ops, but the docs on each op have been relatively sparse. If possible, it would be nice if you could list the count and purpose of the result handles produced by the op, e.g.
```
%fill_op, %first_matmul, %reduce_max, %partial_softmax, %update, %reduce_sum, %reciprocal_sum, %softmax, %scale_acc, %second_matmul
```

Reply:
I plan to send a PR later to make this better, adding documentation and a verifier that checks that the number of results should be consistent.

Reply:
Works for me!